py/__init__.py

```diff
@@ -188,6 +189,7 @@ class PathHolders:
     "List_Long": [PathHolders.ListLongCodec],
     "List_Integer": [PathHolders.ListIntegerCodec],
     "List_UUID": [PathHolders.ListUUIDCodec],
+    "List_List_UUID": [PathHolders.ListListUUIDCodec],
```

Comment:
according to the mono pr ListMultiFrameCodec is needed, so please add it here and to all other clients' files.

Reply:
done